uber/config.py

```diff
@@ -601,7 +607,11 @@ def PAGE(self):
 
     @property
     def PATH(self):
-        return cherrypy.request.path_info.replace(cherrypy.request.path_info.split('/')[-1], '').strip('/')
+        return self._PATH
+
+    @PATH.setter
```

Comment:
Is this so you can set the PATH var for tests? That's neat!

Reply:
Yeah, there were some tests trying to set it at runtime and failing.